ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -394,7 +415,11 @@ private void attachFileToResponse(
 
     try (OutputStream servletOutputStream = response.raw().getOutputStream();
         InputStream resultStream = content.getInputStream()) {
-      IOUtils.copy(resultStream, servletOutputStream);
+      int byteCount = IOUtils.copy(resultStream, servletOutputStream);
+      securityLogger.audit(
+          String.format(
+              "exported metacards: %s from source(s): %s to format: %s with output size: %d",
```

Comment:
>Large streams (over 2GB) will return a bytes copied value of -1 after the copy has completed since the correct number of bytes cannot be returned as an int. For large streams use the [copyLarge(InputStream, OutputStream)](https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/IOUtils.html#copyLarge(java.io.InputStream,java.io.OutputStream)) method.

https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/IOUtils.html#copyLarge(java.io.InputStream,java.io.OutputStream)

just something to think about